### PR TITLE
fix(docs):  local setup with Docker Toolbox for Windows 10 Home

### LINKED ISSUE #36100: Set up freeCodeCamp locally on Windows 10 with Docker Toolbox
I wanted to set up freeCodeCamp locally on Windows 10.
I have followed the [guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-setup-freecodecamp-locally.md) with the Docker build as it is recommended (By the way, the guide is clear and well written).
I've installed the Docker Toolbox as I have Windows 10 Family (v 1803) and I've changed the DOCKER_HOST_LOCATION to the IP adress indicated by docker-machine ip.
Then I've ran `npm run docker:init` but it doesn't work as the screenshot below shows (the two other commands:  `npm run docker:install` and `npm run docker:seed`
do not succeed either).
capture1

A file seems to be indicated as missing but it is present:

capture2

Here is the npm log:

```text
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\tmp\\.nvm\\v10.12.0\\node.exe',
1 verbose cli   'C:\\tmp\\.nvm\\v10.12.0\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'docker:init' ]
2 info using npm@6.4.1
3 info using node@v10.12.0
4 verbose run-script [ 'predocker:init', 'docker:init', 'postdocker:init' ]
5 info lifecycle @freecodecamp/freecodecamp@0.0.1~predocker:init: @freecodecamp/freecodecamp@0.0.1
6 info lifecycle @freecodecamp/freecodecamp@0.0.1~docker:init: @freecodecamp/freecodecamp@0.0.1
7 verbose lifecycle @freecodecamp/freecodecamp@0.0.1~docker:init: unsafe-perm in lifecycle true
8 verbose lifecycle @freecodecamp/freecodecamp@0.0.1~docker:init: PATH: C:\tmp\.nvm\v10.12.0\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;D:\desktop\freeCodeCamp\node_modules\.bin;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\ImageMagick-7.0.7-Q16;C:\Program Files\Intel\iCLS Client\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\MiKTeX 2.9\miktex\bin\x64\;C:\Program Files (x86)\PDFtk Server\bin\;C:\Program Files\Git\cmd;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Calibre2\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\Yarn\bin\;C:\Users\blanc\Miniconda3;C:\Users\blanc\Miniconda3\Library\mingw-w64\bin;C:\Users\blanc\Miniconda3\Library\usr\bin;C:\Users\blanc\Miniconda3\Library\bin;C:\Users\blanc\Miniconda3\Scripts;C:\Users\blanc\AppData\Local\Microsoft\WindowsApps;C:\ffmpeg-20171231-2906363-win64-static\bin;C:\Users\blanc\AppData\Local\Pandoc;C:\Program Files\Zeal;C:\Program Files\Microsoft VS Code\bin;C:\Users\blanc\AppData\Local\Yarn\bin;C:\Users\blanc\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files\Sublime Text 3;D:\Desktop\Programmation\ansifilter-2.9-x64;D:\Desktop\go\bin;C:\Program Files\Docker Toolbox;C:\tmp\.nvm\v10.12.0;C:\Users\blanc\AppData\Roaming\npm
9 verbose lifecycle @freecodecamp/freecodecamp@0.0.1~docker:init: CWD: D:\desktop\freeCodeCamp
10 silly lifecycle @freecodecamp/freecodecamp@0.0.1~docker:init: Args: [ '/d /s /c',
10 silly lifecycle   'docker-compose run -u root --rm freecodecamp bash change_volumes_owner.sh && docker-compose run -u root --rm client bash -c "cd .. && bash change_volumes_owner.sh" && docker-compose run -u root --rm api-server bash -c "cd .. && bash change_volumes_owner.sh"' ]
11 info lifecycle @freecodecamp/freecodecamp@0.0.1~docker:init: Failed to exec docker:init script
12 verbose stack Error: @freecodecamp/freecodecamp@0.0.1 docker:init: `docker-compose run -u root --rm freecodecamp bash change_volumes_owner.sh && docker-compose run -u root --rm client bash -c "cd .. && bash change_volumes_owner.sh" && docker-compose run -u root --rm api-server bash -c "cd .. && bash change_volumes_owner.sh"`
12 verbose stack spawn ENOENT
12 verbose stack     at ChildProcess.<anonymous> (C:\tmp\.nvm\v10.12.0\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:48:18)
12 verbose stack     at ChildProcess.emit (events.js:182:13)
12 verbose stack     at maybeClose (internal/child_process.js:962:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
13 verbose pkgid @freecodecamp/freecodecamp@0.0.1
14 verbose cwd D:\desktop\freeCodeCamp
15 verbose Windows_NT 10.0.17134
16 verbose argv "C:\\tmp\\.nvm\\v10.12.0\\node.exe" "C:\\tmp\\.nvm\\v10.12.0\\node_modules\\npm\\bin\\npm-cli.js" "run" "docker:init"
17 verbose node v10.12.0
18 verbose npm  v6.4.1
19 error file C:\WINDOWS\system32\cmd.exe
20 error code ELIFECYCLE
21 error errno ENOENT
22 error syscall spawn
23 error @freecodecamp/freecodecamp@0.0.1 docker:init: `docker-compose run -u root --rm freecodecamp bash change_volumes_owner.sh && docker-compose run -u root --rm client bash -c "cd .. && bash change_volumes_owner.sh" && docker-compose run -u root --rm api-server bash -c "cd .. && bash change_volumes_owner.sh"`
23 error spawn ENOENT
24 error Failed at the @freecodecamp/freecodecamp@0.0.1 docker:init script.
24 error This is probably not a problem with npm. There is likely additional logging output above.
25 verbose exit [ 1, true ]
```

Someone could help me?

### EDIT: 
I use the second method with npm and mongodb and **it works for me** (except I can't access my portfolio, even when I click on the Preview Custom 404 Page button: but it doesn't matter to me).
Thanks for this huge and useful work!

### PR DESCRIPTION
**Pull request by georgejean**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ ] My changes do not use shortened URLs or affiliate links.

Closes #36100

### CODE REVIEW

**Comment by RandellDawson** [comment]
@georgejean Why did you add the extra spaces and dashes in the table?  It does not appear to change how it displays.

**Comment by georgejean** [comment]
@RandellDawson
Sorry. I have the "Markdown All in One" extension for Visual Studio Code which automatically formats the markdown tables on save. I have fixed this.

**Comment by RandellDawson** [comment]
@raisedadead I may have interpreted your comments on the issue, but I was thinking there was going to actually be a new document containing the supplement information and then link from the how-to-set-up-freecodcamp-locally to this new document.  Instead, we have a link to a forum post?

**Comment by raisedadead** [comment]
Hi @RandellDawson lets give @georgejean some leeway here. I think they are new to contributing on Github.

@georgejean 

- You have closed the pull-request, I am guessing that was just an accident.
- I also noticed that you have two commits, the first one seems to do what is expected, that is add the guide as a new file.
- The second completely removes the file instead of updating the formatting issues.

No worries, you can still re-open the PR and continue working on it. You should totally hit us up on our [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), where we hang out, and should be able to walk you thru this.

**Comment by georgejean** [comment]
@raisedadead 
Indeed, I have closed the pull request by accident.
As RandellDawson pointed out, I put first a simple link to the post where the little guide was (I have not dared to create a new file for that).
But my first commit also has changed the format of some markdown tables so I've had to correct that (hence my second commit).
I try now to reopen the Pull Request with one single commit.

**Comment by raisedadead** [comment]
Ok. That makes sense now. 

We indeed want a new file instead of the issue. You can follow along the current guide for inspiration on the verbiage of the content and add your analysis to it. Then on you can link that new file in the main one as you have already.

Do not worry about multiple commits for now, some one will either squash it or ask for a rebase if needed.

Thanks for the PR.

**Comment by georgejean** [comment]
> Do not worry about multiple commits for now, some one will either squash it or ask for a rebase if needed.

Sorry, I've pushed my change before reading your post.

**Comment by raisedadead** [comment]
> > Do not worry about multiple commits for now, some one will either squash it or ask for a rebase if needed.
> 
> Sorry, I've pushed my change before reading your post.

Do not worry about it. It's fine. Some one will review and give you feedback. You can push additional commits if needed.

**Comment by cmccormack** [comment]
I'm not sure we need a new document for these changes.  There is very little that is unique to the Docker Toolbox setup in the new document, but any time we update information on the original doc it would need to be updated on this doc as well.  I would prefer to see the specific Docker Toolbox instructions be called out inline within the original doc, where appropriate.

**Comment by georgejean** [comment]
So I've put the instructions directly in the guide.
I'm not native English and am not sure of my English, feel free to correct me if you find any spelling errors.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -179,6 +179,30 @@ Keep in mind if you want to use additional services, you'll have to acquire your
 
 **Docker Build:** If using the Docker build and the Docker installation instructed you to use Docker Toolbox (applies to older versions of macOS and Windows), you need to change `DOCKER_HOST_LOCATION` in your `.env` file var to the output from the `docker-machine ip` command. If you use any Docker supported flavor of Linux or if you use Docker Desktop (new versions of macOS and Windows 10) you can leave `DOCKER_HOST_LOCATION` to the default value.
 
+---
+
+*Tip for Windows Home user:*\
+In this case, you have to use Docker Toolbox.
+Your forked freeCodeCamp folder must then be located in the folder `C:\Users\user_name\`.
```
Comment (Nirajn2311) [inline docs/how-to-setup-freecodecamp-locally.md]: I think the "then" is not required here.
```suggestion
Your forked freeCodeCamp folder must be located in the folder `C:\Users\user_name\`. 
```

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -265,6 +289,22 @@ If you visit <http://localhost:3000/explorer> you should see the available APIs.
 
 Congratulations  🎉🎉🎉! You now have a copy of freeCodeCamp's entire learning platform running on your local machine.
 
+---
+
+*Tip for Windows Home user (using Docker Toolbox):*\
+To view the app in your browser, go to the adress indicated:
```
Comment (Nirajn2311) [inline docs/how-to-setup-freecodecamp-locally.md]: Wrong spelling of address
```suggestion
To view the app in your browser, go to the address indicated:
```

**Review by Nirajn2311 — CHANGES_REQUESTED** [review]

File: docs/how-to-setup-freecodecamp-locally.md (hunk 2 above)
Comment (georgejean) [reply docs/how-to-setup-freecodecamp-locally.md]: Thanks!

**Comment by cmccormack** [comment]
@georgejean do you have links to the documentation stating why the cloned freecodecamp repository must be in the user's home directory?  I don't know if this document is the best place to introduce a new concept like symbolic links when they aren't required to work on the repository.  It would be better to link out to an official document from Docker instead.

**Pull request closed without merging**